alacritty.yml

```diff
@@ -449,11 +449,13 @@ key_bindings:
   #- { key: Insert,   mods: Shift,         action: PasteSelection   }
   #- { key: Key0,     mods: Control,       action: ResetFontSize    }
   #- { key: Equals,   mods: Control,       action: IncreaseFontSize }
+  #- { key: Add,      mods: Control,       action: IncreaseFontSize }
   #- { key: Subtract, mods: Control,       action: DecreaseFontSize }
 
   # (macOS only)
   #- { key: Key0,   mods: Command, action: ResetFontSize    }
   #- { key: Equals, mods: Command, action: IncreaseFontSize }
+  #- { key: Add,    mods: Control, action: IncreaseFontSize }
```

Comment:
On macOS we need to use `Command` here.

Reply:
Ope, sorry about that. I just pushed a new commit that should fix this - I also made the same mistake in `bindings.rs`